docs/site/development/development-process.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Handling Tickets (bugs/enhancements)
+---
+
+# Handling Tickets (bugs/enhancements)
+
+### Filing tickets
+
+  1.  You can add set other fields, but don't change the Owner. That way the committee sees the incoming tickets (those marked as Owner: somebody).
+        
+  2.  _**If you really have to fix a bug right away, file it and send a notice to cldr-dev explaining what you are doing and why. Otherwise, file the bug, wait for the next Wednesday meeting for discussion and approval.**_
+    
+
+### Pre-Assessment for Weekly Triage
+
+_There is a pre-assessment of tickets to make the triage flow faster. This pre-assessment should be done off-line prior to every Wednesday in time for the TC triage. DO NOT fix a bug until it's been approved by the TC in triage._
+
+_The triage is a part of Monday and Wednesday meetings._
+
+1.  Ticket comes in, and the default is **Priority**\=assess, **Milestone**\=to-assess
+    
+    1.  If user sets Component (we always should!) automatically assigns the Owner based on the component owner for the pre-assessment. \[Add link to Components and Owner\]
+        
+    2.  _At this point the Owner function is really as an assessor._
+        
+    3.  If user does not select the Component, it remains for the group TC triage with Component=to-assess, Owner=anybody. These tickets will be assessed after all the pre-assessed tickets have been triaged.
+        
+2.  All TC members, look at [To-assess by Owner / Component](https://www.google.com/url?q=https%3A%2F%2Funicode.org%2Fcldr%2Ftrac%2Freport%2F100&sa=D&sntz=1&usg=AOvVaw3RRcS9UWwOlotoRAFsV3fJ) before each triage day.
```

Comment:
You need to resolve links that start with `https://www.google.com/url?q=` first.

Easiest way is to paste into a browser's address bar; the browser will resolve it — in this case, to:

https://cldr.unicode.org/index/bug-reports#TOC-Filing-a-Ticket